app/controllers/placements/pages_controller.rb

```diff
@@ -6,4 +6,10 @@ def start
       content: File.read(Rails.root.join("app/views/placements/pages/landing-page.md")),
     }
   end
+
+  def cookies
+    render locals: {
+      content: File.read(Rails.root.join("app/views/placements/pages/cookies.md")),
+    }
+  end
```

Comment:
For the next static page we implement, I'd be tempted to make this into a generic "show" action.

Something like:

```suggestion
  def show
    page = params[:page]
    render locals: {
      content: File.read(Rails.root.join("app/views/placements/pages/#{page}.md")),
    }
  end
```

and then in the routes:

```ruby
scope module: :pages do
  get :start
  get :cookies, action: :show, page: :cookies
  get :privacy, action: :show, page: :privacy
end
```

Meaning we only need one `show.html.erb` view file for basic static pages.

Of course, the start/landing page is a special case. It needs a custom view because it adds the 'start button' and sidebar links.